certmanager/generate_cert.go

```diff
@@ -66,8 +67,21 @@ type CertOptions struct {
 	IsClient bool
 }
 
-// Size of RSA key to generate.
-const rsaBits = 2048
+const (
+	// Size of RSA key to generate.
+	rsaBits = 2048
+
+	// OID tag values for X.509 SAN field (see https://tools.ietf.org/html/rfc5280#appendix-A.2)
+	tagDNSName = 2
+	tagURI     = 6
+	tarIP      = 7
+
+	// The URI scheme for Istio identities.
+	uriScheme = "istio:"
+)
+
+// See http://www.alvestrand.no/objectid/2.5.29.17.html.
+var oidSubjectAltName = asn1.ObjectIdentifier{2, 5, 29, 17}
```

Comment:
Golang does not allow this to be defined as a const var since the object is not immutable